src/components/modals/TradePreviewModalGP.vue

```diff
@@ -8,14 +8,12 @@
           >
             <span>
               {{ $t('effectivePrice') }}
-            </span>
-            <span
-              v-html="
```

Comment:
This used to have `html` in it... but not anymore - so I've removed it (which also fixed the spacing issue)